virttest/utils_test/utils_nwfilter.py

```diff
@@ -1,10 +1,79 @@
 """
+Virtualization test - network filter related utility functions
 Module simplifying manipulation of XML described at
 http://libvirt.org/formatnwfilter.html
 """
 
+import logging
+
 from virttest.libvirt_xml import base, accessors
 from virttest.libvirt_xml.devices.interface import Filterref
+from virttest import libvirt_xml
+
+# network filter xml set up code BEGIN
+
+
+def set_nwfilter_iface(New_iface,
+                       type_name="network",
+                       source={'network': "default"},
```

Comment:
do not use default network, as it's reserved by libvirt and normally not allowed to be modified by others. 
I think source={} is fine, and add empty check in code if {} is not allowed